### ISSUE TITLE
Make More variants 

### ISSUE THREAD

**Issue by Deottive** [insanum/obsidian_gruvbox#21]
I would love if you made 3 of them for Light and dark like 
1. Dark Hard Gruvbox 
2. Dark Medium Gruvbox
3. Dark Soft Gruvbox [which is the current one]
and the same for Light versions 

Just an Idea

**Comment by insanum** [comment]
I agree this would be nice. How would it work inside Obsidian? Right now there is only dark/light selections.
